### PR TITLE
User interface for Tic Tac Toe Game

### PR DESCRIPTION
**Pull request by anewman15**
In this PR, we created the possible prompts to direct the users of the tic-tac-toe game. They include:

- An outline of the tic-tac-toe board to help the users see the cells.
- Prompts for users to make their moves.
- Prompts to make them sure about their moves.
- Show the updated board.
- Text showing that their move has been updated on the board.
- Text showing that the turn has moved to the other player.
- Text to remind them if the cell they entered is already occupied.
- Text when a player wins the game.

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1,3 +1,70 @@
...
+player2_decision = gets.chomp
+puts "Your decision for #{player2_input} is #{player2_decision}."
+
+# New turn for player 1
+puts "Now it's Player 1's turn. Player 1, please make your move"
+player1_input = gets.chomp
+
+# if user input already occupied
+puts "#{player1_input} is already occupied. Please look at the board and fill another cell."
+# Display board
```
Comment (maelfosso) [inline bin/main.rb]: Please, use a **while/loop**
```rb
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @anewman15 @tGodson 

Nice introduction to this milestone.
We know it's not really easy to clearly understand what it's expected from students at this level, so I'm going to give some indications

1. It's not good to chain puts/gets as you did (read comments). To finish a Tic-Tac-Toe, there are at most 9 moves, but you agree with me that as a software developer you can't write 9 block of instructions doing the same thing. Right? That is why I suggest you to use **while loop**

2. Don't ask the user to confirm his input. On the contrary, you have the check (at the next milestone) if it's valid (is the cell free? is it a number?)

3. There is the expected behavior 
* [x] `[OPTIONAL]` game asks players for their names
* [x] `[REQUIRED]` game tells which player turn it is
* [x] `[REQUIRED]` game asks the player to select from available moves
* [x] `[REQUIRED]` game informs player if the selected move is invalid
* [x] `[REQUIRED]` game displays board after the player move
* [x] `[REQUIRED]` game informs player if the selected move is a winning move
* [x] `[REQUIRED]` game informs player if the selected move is a draw move
* [x] `[REQUIRED]` game repeats all actions for the next player's move

Kindly make the changes and submit a new code review request
Thanks

Mael FOSSO

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Great work! @anewman15 and @tGodson 
You have done well building this, however, a few things are amiss. For one, you should print the board with the current moves after every turn so the user can see and understand what's going on. Second, please consider making the game play simpler, instead of `A1`, a simple `1` will be fine and go a long way for user experience. 

Kindly consider my comments above, make the required changes and request another review.

**Comment by Dandush03** [comment]
# Status: 🟢 Approved 🟢 
## Great work applying previews request,
### Good luck on your next assignment
#### Happy coding

**Pull request merged**